html/en-US/privacypolicy.html

```diff
@@ -22,6 +22,7 @@
       <li>2.1. "Personal Information": for subscription and / or registration for full enjoyment of the tools offered by TEKSTO, users must provide certain Personal Information necessary for their identification, such as contact information, age, location, spoken language and gender, email, and so on. For the acquisition of services, financial information will be requested. This information is provided voluntarily and its truthfulness, timeliness and completeness is the sole responsibility of the users.</li>
       <li>2.2. "Access and Interaction Information": TEKSTO may make use of technologies to collect and store User Access and Interaction Information for the purpose of completing and improving the application database, which may include: a) Information about the user's access device, such as operating system and browser; b) Internet protocol (IP) address with provision of location and time of access of the user; c) Cookies and anonymous identifiers - pieces of information containing a string of characters that identify it, which are stored on the user's device when accessing a site or application - for the purpose of maintaining records and navigation preferences, in order to optimize quality of the service accessed; d) forms of communication of the user, either through e-mail or other means of communication with TEKSTO, these communications may be retained for the processing of requests and improvement of services.</li>
       <li>2.3. For the purposes described in Clause 2.2, TEKSTO reserves the right to use any and all functionality of tools, softwares, applications and analytics services to evaluate user interaction and profile (such as Google Analytics, RD Station, as well as other tools developed by Facebook, Amazon Store and so on). The user is, however, allowed to configure their browser to refuse cookies, or to indicate when a cookie is being sent. However, any features and services of the TEKSTO application may not work correctly if the acceptance of cookies is disabled.</li>
+      <li>2.4. Intelectual property content are not collected to ends of creation of data base or anallisis, although those contents might be acessed by TEKSTO development team, only and exclusively in order to find and fix possible tecnological errors, also known as "bugs", individually of each user, to preserve all the content from any problems. However, all intelectual property of our users is treated as confidential and protected information and it is subject to security measurements to avoid access, use or non allowed disclosure.</li>

```

Comment:
```suggestion
      <li>2.4. Intellectual property content is not collected to ends of creation of database or analysis, although those contents might be accessed by TEKSTO development team, only and exclusively in order to find and fix possible technological errors, also known as "bugs", individually of each user, to preserve all the content from any problems. However, all intellectual property of our users is treated as confidential and protected information and it is subject to security measurements to avoid access, use, or non-allowed disclosure.</li>

```